src/zmalloc.c

```diff
@@ -626,9 +626,54 @@ size_t zmalloc_get_rss(void) {
 
 #if defined(USE_JEMALLOC)
 
-int zmalloc_get_allocator_info(size_t *allocated,
-                               size_t *active,
-                               size_t *resident) {
+#include <assert.h>
```

Comment:
i see in here, we removed this line (assert.h). https://github.com/redis/redis/pull/12539/files#diff-b601b5ff4c0d077e82c78d4059ce3d10de7a54a8965ec9e7b6515b799f930ac8L39

i can replace it with redisassert.h in #13068 if needed.